lib/Models/Subscriber.php

```diff
@@ -783,6 +783,7 @@ static function setRequiredFieldsDefaultValues($data) {
       'first_name' => '',
       'last_name' => '',
       'unsubscribe_token' => Security::generateUnsubscribeToken(self::class),
+      'link_token' => Security::generateRandomString(32),
```

Comment:
I am using random string for new users because I think it's more secure, but I have to keep the current algorithm for existing users because they may already have associated links.